apps/web/ee/lib/workflows/reminders/emailReminderManager.ts

```diff
@@ -0,0 +1,145 @@
+import {
+  TimeUnit,
+  WorkflowTriggerEvents,
+  WorkflowTemplates,
+  WorkflowActions,
+  WorkflowMethods,
+} from "@prisma/client";
+import client from "@sendgrid/client";
+import sgMail from "@sendgrid/mail";
+
+import dayjs from "@calcom/dayjs";
+import { sendWorkflowReminderEmail } from "@calcom/emails";
+import prisma from "@calcom/prisma";
+import { BookingInfo, timeUnitLowerCase } from "@ee/lib/workflows/reminders/smsReminderManager";
+import emailReminderTemplate from "@ee/lib/workflows/reminders/templates/emailReminderTemplate";
+
+let sendgridAPIKey, senderEmail: string;
+
+if (process.env.SENDGRID_API_KEY) {
+  sendgridAPIKey = process.env.SENDGRID_API_KEY as string;
+  senderEmail = process.env.SENDGRID_EMAIL as string;
+
+  sgMail.setApiKey(sendgridAPIKey);
+  client.setApiKey(sendgridAPIKey);
+}
+
+export const scheduleEmailReminder = async (
+  evt: BookingInfo,
+  triggerEvent: WorkflowTriggerEvents,
+  action: WorkflowActions,
+  timeBefore: {
+    time: number | null;
+    timeUnit: TimeUnit | null;
+  },
+  sendTo: string,
+  emailSubject: string,
+  emailBody: string,
+  workflowStepId: number,
+  template: WorkflowTemplates
+) => {
+  const { startTime } = evt;
+  const uid = evt.uid as string;
+  const currentDate = dayjs();
+  const timeUnit: timeUnitLowerCase | undefined =
+    timeBefore.timeUnit?.toLocaleLowerCase() as timeUnitLowerCase;
+  const scheduledDate =
+    timeBefore.time && timeUnit ? dayjs(startTime).subtract(timeBefore.time, timeUnit) : null;
+
+  if (!process.env.SENDGRID_API_KEY || !process.env.SENDGRID_EMAIL) return;
```

Comment:
Perhaps add a console.warning that a remainder was attempted to be sent without the proper api env vars set